src/java.base/share/classes/javax/crypto/CipherSpi.java

```diff
@@ -278,22 +278,23 @@ protected abstract void engineSetPadding(String padding)
      * <p> This is useful in the context of password-based encryption or
      * decryption, where the IV is derived from a user-provided passphrase.
      *
-     * @return the initialization vector in a new buffer, or null if the
-     * underlying algorithm does not use an IV, or if the IV has not yet
-     * been set.
+     * @return the initialization vector in a new buffer, or {@code null} if the
+     * algorithm does not use an IV, or if the IV has not yet
+     * been set
      */
     protected abstract byte[] engineGetIV();
 
     /**
      * Returns the parameters used with this cipher.
      *
      * <p>The returned parameters may be the same that were used to initialize
-     * this cipher, or may contain a combination of default and random
-     * parameter values used by the underlying cipher implementation if this
-     * cipher requires algorithm parameters but was not initialized with any.
+     * this cipher, or may contain additional default or random parameter
+     * values used by the underlying cipher implementation. If the required
+     * parameters were not supplied and can be generated by the cipher, the
+     * generated parameters will be returned. Otherwise, {@code null} is
```

Comment:
Minor wording nit - change "will be" to "are" to be consistent with other wording which uses the present tense. Same comment applies to `Cipher.getParameters`.

Reply:
Yes, will fix the nit.